### DISCUSSION TITLE
New tab for "best comments" and "highlights"

### DISCUSSION THREAD

**Discussion by trySometngNew** [SimonHalvdansson/Harmonic-HN#106] [Ideas]
Hi @SimonHalvdansson,
Can you include, if api endpoint available and if possible, best comments and highlights pages of hn? They are mentioned on lists page of hn. I would definitely be very interested to browse those hn pages.

**HN lists** : https://news.ycombinator.com/lists
**Best Comments** : https://news.ycombinator.com/bestcomments
**Highlights** : https://news.ycombinator.com/bestcomments
**Active** : https://news.ycombinator.com/active

Not very interested in active page, but may be useful for everyone else?

Regards, 
Sourabh

**Comment by SimonHalvdansson** [comment]
Hey this is actually pretty good. It felt a bit ad-hoc adding just Active (there was an issue for it) but with all of these (hidden behind a toggle in the settings I think), it's more coherent. I'm opening https://github.com/SimonHalvdansson/Harmonic-HN/issues/313 for this
